tools/nl/loadtest/load.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import concurrent.futures
+import csv
+import json
+
+from absl import app
+from absl import flags
+import requests
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('parallel_requests', 0,
+                     'Number of requests to run in parallel')
+flags.DEFINE_integer('total_requests', 0, 'Total number of requests to run')
```

Comment:
Why are the defaults 0?  Can we pick something reasonable?

Reply:
I originally wanted user to have to input their own numbers, but I've updated it so that if no parallel requests or total requests received, just run the list of queryset queries once in series.